packages/rpc-graphql/src/loaders/block.ts

```diff
@@ -30,7 +30,7 @@ function normalizeArgs({
 /* Load a block from the RPC, transform it, then return it */
 async function loadBlock(rpc: Rpc, { slot, ...config }: ReturnType<typeof normalizeArgs>) {
     const block = await rpc
-        .getBlock(slot, config as unknown as Parameters<SolanaRpcMethods['getBlock']>[1])
+        .getBlock(slot, config as unknown as Parameters<GetBlockApi['getBlock']>[1])
```

Comment:
Dang. This is because TypeScript can't read all the way through all the overloads of `getBlock`, hey?

Rather than to cast this stuff through `unknown` I'd maybe be inclined to `// @ts-expect-error {LINK TO GITHUB ISSUE}` so these don't get forgotten forever.